deepchem/feat/molecule_featurizers/conformer_featurizer.py

```diff
@@ -0,0 +1,197 @@
+import numpy as np
+from rdkit import Chem
+from rdkit.Chem import AllChem
+from deepchem.feat.graph_data import GraphData
+from deepchem.feat import MolecularFeaturizer
+
+# similar to snap featurizer. both taken from Open Graph Benchmark (OGB) github.com/snap-stanford/ogb
+allowable_features = {
+    'possible_atomic_num_list': list(range(1, 119)) + ['misc'],  # type: ignore
+    'possible_chirality_list': [
+        'CHI_UNSPECIFIED', 'CHI_TETRAHEDRAL_CW', 'CHI_TETRAHEDRAL_CCW',
+        'CHI_OTHER', 'misc'
+    ],
+    'possible_degree_list': [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 'misc'],
+    'possible_formal_charge_list': [
+        -5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5, 'misc'
+    ],
+    'possible_numH_list': [0, 1, 2, 3, 4, 5, 6, 7, 8, 'misc'],
+    'possible_number_radical_e_list': [0, 1, 2, 3, 4, 'misc'],
+    'possible_hybridization_list': [
+        'SP', 'SP2', 'SP3', 'SP3D', 'SP3D2', 'misc'
+    ],
+    'possible_is_aromatic_list': [False, True],
+    'possible_is_in_ring_list': [False, True],
+    'possible_bond_type_list': [
+        'SINGLE', 'DOUBLE', 'TRIPLE', 'AROMATIC', 'misc'
+    ],
+    'possible_bond_stereo_list': [
+        'STEREONONE',
+        'STEREOZ',
+        'STEREOE',
+        'STEREOCIS',
+        'STEREOTRANS',
+        'STEREOANY',
+    ],
+    'possible_is_conjugated_list': [False, True],
+}
+
+full_atom_feature_dims = list(
+    map(
+        len,  # type: ignore
+        [
+            allowable_features['possible_atomic_num_list'],
+            allowable_features['possible_chirality_list'],
+            allowable_features['possible_degree_list'],
+            allowable_features['possible_formal_charge_list'],
+            allowable_features['possible_numH_list'],
+            allowable_features['possible_number_radical_e_list'],
+            allowable_features['possible_hybridization_list'],
+            allowable_features['possible_is_aromatic_list'],
+            allowable_features['possible_is_in_ring_list']
+        ]))
+
+full_bond_feature_dims = list(
+    map(
+        len,  # type: ignore
+        [
+            allowable_features['possible_bond_type_list'],
+            allowable_features['possible_bond_stereo_list'],
+            allowable_features['possible_is_conjugated_list']
+        ]))
+
+
+def safe_index(feature_list, e):
+    """
+    Return index of element e in list l. If e is not present, return the last index
+    """
+    try:
+        return feature_list.index(e)
+    except ValueError:
+        return len(feature_list) - 1
+
+
+class RDKitConformerFeaturizer(MolecularFeaturizer):
+    """
+    A featurizer that featurizes an RDKit mol object as a GraphData object with 3D coordinates.
```

Comment:
Can you add more details about the conformer generation algorithm and a link to the RDKit docs?

Reply:
done